src/JsonLdWriter.php

```diff
@@ -287,10 +287,14 @@ protected function convertImageToBinary(array|string $value): mixed
             } elseif (array_key_exists('@type', $value)) {
                 // Some other form of image like a Barcode and ImageObjectSnapshot.
                 // See: https://schema.org/ImageObject
-                $this->logger->warning(
-                    "Images of type ". $value['@type']
-                    . " not supported. Image will not be converted."
-                );
+                if(empty($value['@type'])){
+                    $this->logger->warning('@type is empty');
```

Comment:
AFAICS this means that we will not convert the image. Please add that to the warning as well.